utils/swift_build_support/swift_build_support/cmake.py

```diff
@@ -177,3 +181,98 @@ def build_args(self):
                            '-jobs', str(jobs)]
 
         return build_args
+
+    # Determine the version of the installed CMake binary.
+    def installed_cmake_version(self, cmake_binary):
+        version = shell.capture([cmake_binary, '--version'], dry_run=False,
+                                    echo=True, optional=True)
+        if version is None:
+            return (0, 0, 0)
+
+        print(version)
+        x = re.findall(r'cmake version (\d+)\.(\d+)\.(\d+)', version.rstrip())
+        if len(x) == 1:
+            (c_major, c_minor, c_patch) = map(int, x[0])
+
+        return (c_major, c_minor, c_patch)
```

Comment:
In the case that the findall fails, this is going to fail due to use of undefined variables.